app/Policies/UserPolicy.php

```diff
@@ -37,7 +37,7 @@ public function before(?User $user, $ability)
 	 */
 	public function isAdmin(?User $user): bool
 	{
-		return $user?->id === 0;
+		return $user?->may_administrate === true;
 	}
```

Comment:
Should actually not exist at all.

Reply:
But you still need a basic `admin` check for some parts. e.g. access to full settings.

Reply:
This should only happen inside the policies. Outside the policy there should no need to check this, because code outside the policies should query the rights as determined by the policies. And within the policies it is fine to query the user's capability directly, i.e. use `$user->may_administrate`.